src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/CustomConverterTests/CustomConverterTests.BadConverters.cs

```diff
@@ -419,5 +419,82 @@ public override void Write(Utf8JsonWriter writer, ClassWithConverterWithoutPubli
                 throw new NotImplementedException();
             }
         }
+
+
+        [Fact]
+        public static void BadDoubleConverter_Serialization()
```

Comment:
nit: can we have comments or update the test method and/or test type names to indicate whey the converters are "bad"? (Faulty `CanConvert(Type)` implementations...)